website/docs/r/org_vdc_access_control.html.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: org_vdc_access_control"
+sidebar_current: "docs-vcd-resource-org-vdc-access-control"
+description: |-
+Provides a VMware Cloud Director Org Vdc access control resource. This can be
+used to share VDC across users or groups.
+---
+
+# vcd\_org\_vdc\_access\_control
+
+Provides a VMware Cloud Director Org Vdc access control resource. This can be
+used to share VDC across users and/or groups.
+
+Supported in provider *v3.7+*
+
+-> **Note:** This resource requires either system or org administrator privileges.
+
+## Example Usage
+
+```hcl
+resource "vcd_org_vdc_access_control" "my_access_control" {
+  org                  = "my-org"
+  vdc                  = "my-vdc"
+  shared_with_everyone = false
+
+  shared_with {
+    user_id      = vcd_org_user.my-user.id
+    access_level = "ReadOnly"
+  }
+
+  shared_with {
+    user_id      = vcd_org_user.my-user2.id
```

Comment:
Maybe let's add to the example the data sources of how to get these users?

Reply:
Added user datasource for convenience!